src/Tribe/Utils/Element_Attributes.php

```diff
@@ -0,0 +1,228 @@
+<?php
+namespace Tribe\Utils;
+
+/**
+ * Class Element_Attributes to handle HTML attributes for elements.
+ *
+ * @since  TBD
+ *
+ * @package Tribe\Utils
+ */
+class Element_Attributes {
+	/**
+	 * Store the results of parsing the attributes.
+	 *
+	 * @since  TBD
+	 *
+	 * @var array
+	 */
+	protected $results = [];
+
+	/**
+	 * Stores the arguments passed.
+	 *
+	 * @since  TBD
+	 *
+	 * @var array
+	 */
+	protected $arguments = [];
+
+	/**
+	 * Setups an instance of Element Attributes.
+	 *
+	 * @since  TBD
+	 *
+	 * @return void
+	 */
+	public function __construct() {
```

Comment:
This will not inform about what the method does and the method cannot be called without, at the very least, one argument.

```suggestion
	public function __construct( $arg_1 ) {
```